.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -6,14 +6,29 @@ about: Something unexpected happen? Report an error, crash, or an incorrect beha
 
 Hello!
 
-Thank you for opening an issue.  Please note that we try to keep the Terraform MongoDB Atlas Provider issue tracker reserved for bug reports.  Please ensure you check open and closed issues first to ensure your issue hasn't already been reported (if it has been reported add a reaction, i.e. +1, to the issue).  
+Thank you for opening an issue. **These GitHub issues** are only for community reporting and assistance and as such we don't have a guaranteed SLA.
 
-If your issue relates to Terraform itself, please open it in the Terraform repository https://github.com/hashicorp/terraform/issues.
+**If your issue relates to Terraform itself**, please open it in the Terraform repository https://github.com/hashicorp/terraform/issues.
 
-If you have an active MongoDB Atlas Support contract, the best way for us to assist you with the Terraform MongoDB Atlas Provider is through a [support ticket](https://support.mongodb.com/).
+**If you have an active MongoDB Atlas Support contract**, the best way for us to assist you with the Terraform MongoDB Atlas Provider is through a [support ticket](https://support.mongodb.com/).
+
+**Please note:** In order for us to provide the best experience in supporting our customers, we kindly ask to make sure that all the following sections are correctly filled and with all the required information. Our support will **prioritise** issues that contain **all the required** information by using the **"one-click reproducible issues" principle**.
+
+**Please also note:** We try to keep the Terraform MongoDB Atlas Provider issue tracker **reserved for bug reports**. Please ensure you **check open and closed issues first** to ensure your issue hasn't already been reported (if it has been reported add a reaction, i.e. +1, to the issue).
+
+### Guidelines
+
+In order to follow the **"one-click reproducible issues" principle**, please follow these guidelines:
+* We should be able to run `terraform plan` and `terraform apply` **with no errors**.
+  * Configurations that return errors when executed will be de-prioritised in favour of the ones that succeed.
+* Share your configuration by **using variables**. Create a `variables.tf` file and share its content.
+* Before opening an issue, you have to try to specifically isolate it to **Terraform MongoDB Atlas** provider by **removing as many dependencies** as possible. If the issue only happens with other dependencies, then:
+  * If other terraform providers are required, please make sure you also include those. Same "one-click reproducible issue" principle applies.
+  * If external components are required to replicate it, please make sure you also provides instructions on those parts.
+
+* If the issue requires **changes from the UI**, please document them preferably with screenshots
+* If the issue involves **version changes** (e.g. `run plan with version 1.k.x then re-run with version 1.k.y`), please make sure you report the detailed sequence on commands to execute.
```

Comment:
this list is missing the following info:
              - Terraform Atlas provider version used to reproduce the issue
              - Terraform version used to reproduce the issue
              - Confirmation if Terraform OSS, Terraform Cloud, or Terraform Enterprise deployment

Reply:
Regarding:
```
- Terraform Atlas provider version used to reproduce the issue
- Terraform version used to reproduce the issue
```
maybe we can clarify that the `version.tf` file is also needed (i.e. the `provider` specification). 
I think I specified somewhere else that the *whole* configuration is needed in order for us to be _one-click-away_ from reproducing the issue; but I'll clarify that.

```
- Confirmation if Terraform OSS, Terraform Cloud, or Terraform Enterprise deployment
```
Ok sure, this one I missed it.

Reply:
Changes updated. Let me know if that's ok with you.